dbschema/partner.esdl

```diff
@@ -0,0 +1,38 @@
+module default {
+  type Partner extending Resource, Project::ContextAware, Mixin::Pinnable, Mixin::Taggable {
```

Comment:
We should automatically create a project context for this. Just like we do here
https://github.com/SeedCompany/cord-api-v3/blob/9f47163ab06438265a7e8dff549522ab74bcc596/dbschema/language.esdl#L82-L84

Reply:
So we're saying that a `Partner` has a sensitivity based upon all the `Project`s it's connected to?  I'm not sure I realized that.

Also, what does the `isMember` gain us here for a `Partner` from a business perspective?  Is this just so that on a Partner's page we can see which Projects they are a part of?

Reply:
The `isMember` piece is a little fuzzy, but I get the idea; thanks.